cscs-checks/cuda/cuda_memtest_check.py

```diff
@@ -0,0 +1,46 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class FlexibleCudaMemtest(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.valid_systems = ['daint:gpu', 'dom:gpu']
+        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
+                                    'PrgEnv-intel', 'PrgEnv-pgi']
+        self.descr = 'Flexible Cuda Memtest'
+        self.executable = 'osu_alltoall'
+        self.maintainers = ['TM', 'VK']
+        self.tags = {'maintenance'}
+        self.num_tasks_per_node = 1
+        self.num_tasks = 0
+        self.num_gpus_per_node = 1
+        self.modules = ['cudatoolkit']
+        self.sourcesdir = None
+        src_url = ('https://downloads.sourceforge.net/project/cudagpumemtest/'
+                   'cuda_memtest-1.2.3.tar.gz')
+        self.prebuild_cmd = ['wget %s' % src_url,
+                             'tar -xzf cuda_memtest-1.2.3.tar.gz '
+                             '--strip-components=1']
```

Comment:
If you format the list as follows, the second command might fit in a single line:

```python
self.prebuild_cmd = [
    'wget ...'
    'tar ...'
]
```